backend/docs/db/schema.sql

```diff
@@ -468,12 +468,17 @@ CREATE VIEW public.sequence_entries_view AS
     sepd.warnings,
         CASE
             WHEN (se.released_at IS NOT NULL) THEN 'APPROVED_FOR_RELEASE'::text
-            WHEN se.is_revocation THEN 'AWAITING_APPROVAL'::text
+            WHEN se.is_revocation THEN 'PROCESSED'::text
             WHEN (sepd.processing_status = 'IN_PROCESSING'::text) THEN 'IN_PROCESSING'::text
-            WHEN (sepd.processing_status = 'HAS_ERRORS'::text) THEN 'HAS_ERRORS'::text
-            WHEN (sepd.processing_status = 'FINISHED'::text) THEN 'AWAITING_APPROVAL'::text
+            WHEN (sepd.processing_status = 'PROCESSED'::text) THEN 'PROCESSED'::text
             ELSE 'RECEIVED'::text
-        END AS status
+        END AS status,
+        CASE
+            WHEN (sepd.processing_status = 'IN_PROCESSING'::text) THEN NULL::text
+            WHEN ((sepd.errors IS NOT NULL) AND (jsonb_array_length(sepd.errors) > 0)) THEN 'HAS_ERRORS'::text
+            WHEN ((sepd.warnings IS NOT NULL) AND (jsonb_array_length(sepd.warnings) > 0)) THEN 'HAS_WARNINGS'::text
+            ELSE 'NO_ISSUES'::text
```

Comment:
Not blocking, should probably create issue but noticed here: it's rather strange we use a text column to store what is in reality an enum. We should use enums here: https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
I've made a separate issue regarding possible schema improvements: https://github.com/loculus-project/loculus/issues/3201

Reply:
thanks for picking this up and creating the ticket! I'm marking this as resolved here, as we won't do anything in this PR.